pkg/config/node/minimum_kubelet_version.go

```diff
@@ -0,0 +1,80 @@
+package node
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/blang/semver/v4"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+// An error to be returned when kubelet version is lower than specified minimum kubelet version.
+// Used to differentiate between a parsing failure, and a failure because the kubelet is out of date.
+var ErrKubeletOutdated = fmt.Errorf("kubelet version is outdated")
```

Comment:
In the missing test case, we could ensure that the error can be unwrapped to the sentinel error.

Reply:
https://github.com/openshift/library-go/pull/1831/files#diff-46d78ef74ff7e0efcd85f615801ad28063fd5732d099050c47db754940e16f91R193-R195 does with https://github.com/openshift/library-go/pull/1831/files#diff-46d78ef74ff7e0efcd85f615801ad28063fd5732d099050c47db754940e16f91R84